Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -221,6 +221,54 @@ class PaymentSheetUITest: XCTestCase {
         XCTAssertNotNil(successText.label.range(of: "Your order is confirmed!"))
     }
 
+    func testDeferredPaymentIntent() {
+        loadPlayground(
+            app,
+            settings: [
+                "init_mode": "Deferred"
+            ]
+        )
+
+        app.buttons["Checkout (Complete)"].tap()
+        let payButton = app.buttons["Pay $10.00"]
+        XCTAssertTrue(payButton.waitForExistence(timeout: 10.0))
+
+        // TODO(porter) Finish test when we can confirm server side
+    }
+
+    func testDeferredSetupIntent() {
+        loadPlayground(
+            app,
+            settings: [
+                "init_mode": "Deferred",
+                "mode": "Setup",
+            ]
+        )
+
+        app.buttons["Checkout (Complete)"].tap()
+        let setupButton = app.buttons["Set up"]
+        XCTAssertTrue(setupButton.waitForExistence(timeout: 10.0))
+
+        // TODO(porter) Finish test when we can confirm server side
+    }
+
+    func testDeferredPaymentIntent_FlowController() {
```

Comment:
why these particular tests and not others e.g. `testDeferredSetupIntent_FlowController`?

in general I haven't thought about how to test the deferred flows, eager to hear your thoughts!

Reply:
No rhyme or reason really, I haven't given much thought to how we will test the deferred flows. I think that will become more clear when we understand the nuance of server side confirmation better.

For now I'm considering these stubs to make writing the real tests easy while still providing some value. I'm sure we will end up adding more tests as the server side confirmation stuff becomes clearer.

I did add `testDeferredSetupIntent_FlowController` for consistency.

Reply:
Ok, makes sense